### ISSUE TITLE
remove pystyle

### ISSUE THREAD

**Issue by CSM-BlueRed** [bastienbournas/w4spHunter#1]
pystyle has not been infected by wasp, and its a hacker that do this in pystyle, we removed the suspect line.

**Comment by bastienbournas** [comment]
Hi, what do you mean ? pystyle is referenced as malicious in the different articles about this:

- https://www.bleepingcomputer.com/news/security/dozens-of-pypi-packages-caught-dropping-w4sp-info-stealing-malware/
- https://blog.phylum.io/phylum-discovers-dozens-more-pypi-packages-attempting-to-deliver-w4sp-stealer-in-ongoing-supply-chain-attack
- https://thehackernews.com/2022/11/researchers-uncover-29-malicious-pypi.html

**Comment by CSM-BlueRed** [comment]
yeah i know, cause a hacker put a line with a module nammed _algoritmic (not the exact name) and we removed it, you can check the pypi logs

**Comment by bastienbournas** [comment]
Ok, so the package could still be vulnerable if people did not update ?

**Comment by CSM-BlueRed** [comment]
if people update pystyle, nothing happend, and if people doesnt update pystyle, nothing happend too

**Comment by bastienbournas** [comment]
So at some point in time there was a version of pystyle which had this vulnerability right ? I can add a specific case for version number, but I think the tool has to report something for pystyle if it was vulnerable

**Comment by CSM-BlueRed** [comment]
yes if you want
